recipes-support/lvm2/lvm2_%.bbappend

```diff
@@ -22,6 +26,10 @@ do_install_append() {
     install -m 0644 ${WORKDIR}/volatiles.99_cryptsetup ${D}${sysconfdir}/default/volatiles/99_cryptsetup
 }
 
+ldconfig_postinst_fragment() {
+}
+ALTERNATIVE_lvm2_remove = "lvm.conf"
```

Comment:
That should not be a problem with the following:
```
PACKAGES =+ "${PN}-conf"
RRECOMMENDS_${PN}_append += "${PN}-conf"

FILES_${PN}-conf = " \
    ${sysconfdir}/lvm/lvm.conf \
"
```
I do not see `ALTERNATIVE` definitions in `meta-openembedded` either, what reason triggered this change?

Reply:
This issue is surfaced when multilib is enabled. I haven't tested your proposed alternative here - does it coexist ok with the `lvm2-conf-initramfs` OpenXT recipe?

Reply:
`lvm2-conf-initramfs` was introduced for the above sleigh of hand (https://github.com/OpenXT/xenclient-oe/commit/8950739df & https://github.com/OpenXT/xenclient-oe/commit/104ddd73389740513d88ea3ea1b6ed918ae151bc)

Reply:
ok, I now have an alternative proposal for this that doesn't do the above and still builds successfully: the lvm2 recipe has some multilib-aware logic and a small similar piece needs adding to the `lvm2-conf-initramfs` recipe:
- add `inherit multilib_script`
- add `MULTILIB_SCRIPTS = "${PN}:${sysconfdir}/lvm/lvm.conf"`